### PR TITLE
Update fonts

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

**Comment by Leeseli** [comment]
Hi, here are the texts that need to be updated:

- [ ] create strategy page- "next step" button needs to be BW 
image

- [ ] my strategy page- "create your first.." needs to be BW 
image

- [ ] trade page - "buy" and "sell" buttons, text font should be 16px and bold.
image

thank you!

File: src/components/simulator/result/SimResultSummary.tsx [hunk 1]
```diff
@@ -79,10 +79,10 @@ export const SimResultSummary = ({
...
                 sellBudget: state.sell.budget,
               }}
-              className={cn(
-                buttonStyles({ variant: 'success', size: 'md' }),
-                'whitespace-nowrap'
-              )}
+              className={`
+                ${buttonStyles({ variant: 'success', size: 'md' })}
+                whitespace-nowrap
+              `}
```
Comment (tiagofilipenunes) [inline src/components/simulator/result/SimResultSummary.tsx:85]: buttonStyles doesn't apply white-space, so it's fine not to use cn()

File: src/components/strategies/create/CreateStrategyTypeMenu.tsx [hunk 2]
```diff
@@ -183,12 +183,14 @@ export const CreateStrategyTypeMenu: FC<UseStrategyCreateReturn> = ({
...
-            : 'pointer-events-none cursor-not-allowed opacity-40'
-        )}
+        className={`
+          ${buttonStyles({ variant: 'success', fullWidth: true, size: 'lg' })}
+          ${
+            selectedStrategySettings
+              ? ''
+              : 'pointer-events-none cursor-not-allowed opacity-40'
+          }
+        `}
```
Comment (tiagofilipenunes) [inline src/components/strategies/create/CreateStrategyTypeMenu.tsx:193]: buttonStyles applies the same opacity and cursor-not-allowed, so I think it's fine not to use cn(), please confirm that's intended

File: src/components/core/menu/mainMenu/MainMenuRightWallet.tsx [hunk 3]
```diff
@@ -111,9 +111,7 @@ export const MainMenuRightWallet: FC = () => {
             data-testid="user-wallet"
           >
             {buttonIcon}
-            <span className={debug.debugState.isE2E ? 'font-mono' : ''}>
-              {buttonText}
-            </span>
+            <span>{buttonText}</span>
```
Comment (tiagofilipenunes) [inline src/components/core/menu/mainMenu/MainMenuRightWallet.tsx:114]: Assuming removing font-mono here won't affect e2e screenshots

File: e2e/screenshots/strategy/recurring/Recurring limit range/editPrices/form.jpg
Comment (tiagofilipenunes) [inline e2e/screenshots/strategy/recurring/Recurring limit range/editPrices/form.jpg:1]: Strategy budget distribution text is wrapping here

**Review by tiagofilipenunes — CHANGES_REQUESTED** [review]
Looks beautiful! 🎉

Observed:
- Ethereum Network text in the user wallet dropdown now wrapping
Before:

Screenshot 2024-04-18 at 15 33 00

After:
Screenshot 2024-04-18 at 15 32 54

- Titles all appropriately assigned, WarningMessageWithIcon applied everywhere 🤌
- Clearer to read numbers in charts (simulator, strategy block and overlapping graph)

Places where the new fonts weren't applied:
- [x] Browser default font being used in the trade page on the tooltip when hovering the depth chart
- [ ] Browser default font being used in the tooltip when hovering the portfolio pie chart

File: e2e/screenshots/strategy/recurring/Recurring limit range/editPrices/form.jpg
Comment (GrandSchtroumpf) [reply e2e/screenshots/strategy/recurring/Recurring limit range/editPrices/form.jpg:1]: Lee decided to keep it in two lines

**Review by tiagofilipenunes — APPROVED** [review]

**Review by ashachaf — APPROVED** [review]

**Pull request merged**